pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -80,27 +80,29 @@ func (sv *openShiftClusterStaticValidator) validateProperties(path string, p *Op
 	default:
 		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, path+".provisioningState", "The provided provisioning state '%s' is invalid.", p.ProvisioningState)
 	}
+	if isCreate {
```

Comment:
* unit test coverage needed
* also want to see stronger unit tests for delta validation, e.g. cases where the length of WorkerProfiles changes

Reply:
It is still a draft ;)